Modules/Relationships/test/Relationships.Application.Tests/TestHelpers/TestData.cs

```diff
@@ -48,22 +48,15 @@ public static Relationship CreateTerminatedRelationship(IdentityAddress activeId
         return relationship;
     }
 
-    public static Relationship CreateRelationshipWithRequestedReactivation(IdentityAddress from, IdentityAddress to, IdentityAddress reactivationRequestedBy)
+    public static Relationship CreateRelationshipWithRequestedReactivation(IdentityAddress? from = null, IdentityAddress? to = null, IdentityAddress? reactivationRequestedBy = null)
```

Comment:
Why did you make the parameters optional? I don't see any place where you don't pass anything.

Reply:
refactoring gone wrong...